libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
@majormoses yes, peers won't list a node that was reset as an online peer. Are you sure that they also won't list it as an offline one? I see the same assumption made over and over, that if a node is not online it must not be a cluster member any more. That's not how things work, as explained in https://github.com/rabbitmq/chef-cookbook/issues/432#issuecomment-295018420.